public/components/cards/23.html

```diff
@@ -23,7 +23,7 @@
       <div>
         <dt class="sr-only">
           Address
-        </dd>
+        </dt>
```

Comment:
Following the same layout on the component, I believe this is meant to be a closing `</dt>`.